components/post_view/post_info/post_info.jsx

```diff
@@ -4,22 +4,18 @@
 import PropTypes from 'prop-types';
```

Comment:
I think you are including here unrelated refactor of the post_info. I'm not against the refactor itself, looks like better code, but is harder to evaluate both at the same time. Would be possible to include only the changes related to the new E2E testing, and move the refactor of the post_info to another PR. If is too much work, I can review it anyway. But I think they independent changes and must be in different PRs.

Reply:
Refactor is now submitted separately - https://github.com/mattermost/mattermost-webapp/pull/2367.  This PR is on top of that.  You may find the changes specifically to E2E at https://github.com/mattermost/mattermost-webapp/pull/2355/commits/78bd50b758b820649421392e4686323b69b8a1ab.

Reply:
Thank!